### ISSUE TITLE
more info at  Attempt to reconnect  dialog

### ISSUE THREAD

**Issue by ercanal** [TigerVNC/tigervnc#1627]
It would be great if previously connected username and hostname with display id is shown at the below dialog.  I have 2-3 vnc sessions with different users. I need to try each of them or reconnect again.

![image](https://user-images.githubusercontent.com/10163864/235657984-da3a5cc3-dad3-408b-9d62-c2c1a3f215a4.png)

**Comment by CendioOssman** [comment]
Yeah, I think we should at least include the same information that was specified in the connection dialog, i.e. host and display/port. Authentication details could be more difficult as they are handled in a more complex way.

**Comment by htInEdin** [comment]
Yes _please_!  I regularly have 4 connections, and they drop with some regularity, and seeing the hostname and port in any pop-up, but particularly the lost connection one, would be very helpful!

**Comment by ercanal** [comment]
![image](https://github.com/TigerVNC/tigervnc/assets/10163864/84dfef12-ea16-4b1a-8adc-261cfcdc026e)
For example this windows has already has this information. 
Just need to change the other one similar to this one.

**Comment by CendioOssman** [comment]
Unfortunately, it's that particular error message that happens to include the host. So it's not just a matter of swapping things around.

(we also seem to be in dire need of some wrapping of long lines)

**Comment by ercanal** [comment]
Is it possible to implement this feature with new added "keep password for reconnect" option.
